lib/Serialization/ModuleFile.h

```diff
@@ -108,7 +108,7 @@ class ModuleFile
   /// Represents another module that has been imported as a dependency.
   class Dependency {
   public:
-    ModuleDecl::ImportedModule Import = {};
+	llvm::Optional<ModuleDecl::ImportedModule> Import = llvm::Optional<ModuleDecl::ImportedModule>();
```

Comment:
Couple of things:

1. You can use `llvm::None` instead of `llvm::Optional<ModuleDecl::ImportedModule>()` 🙂.
2. You can dedent by 2 spaces (since that was the original indentation).

Reply:
Resolved in 1d625d8